### PR TITLE
kn-admin domain unset command

### PR DESCRIPTION
**Pull request by ZhuangYuZY**
For issue https://github.com/knative/client-contrib/issues/29:

Now we have kn admin domain set to set custom-domain. But we are missing to unset the custom domain if it is not used or incorrect.

unset Knative route domain for service

For example:
# To unset a route domain
kn admin domain unset --custom-domain mydomain.com

Usage:
  kn admin domain unset [flags]

Flags:
  -d, --custom-domain string  Custom domain to unset
  -h, --help                   help for unset

Global Flags:
      --config string   config file (default is $HOME/.config/kn/plugins/admin.yaml)

### CODE REVIEW

**Comment by zhanggbj** [comment]
Hi @rhuss, I remember you have a contact to help us join Knative Org, would you please help to add @ZhuangYuZY in Knative Org, or any guide we can follow? Thanks!

File: plugins/admin/pkg/command/domain/domain.go [hunk 1]
```diff
@@ -24,11 +24,14 @@ func NewDomainCmd(p *pkg.AdminParams) *cobra.Command {
 	var domainCmd = &cobra.Command{
 		Use:   "domain",
 		Short: "Manage route domain",
-		Long: `Set default route domain or custom route domain for Service with selectors. For example:
+		Long: `Set default route domain or custom route domain for Service with selectors, also support unset route domain. For example:
```
Comment (zhanggbj) [inline plugins/admin/pkg/command/domain/domain.go:27]: Hi @ZhuangYuZY ,

As the long sentence is hard to read on console, how about we shorten this to 
`Manage default route domain or custom route domain for Service with selectors`
or 
`Set or unset default route domain or custom route domain for Service with selectors`?

Personally I prefer `Manage` as we'll also have a `domain list` later.

File: plugins/admin/pkg/command/domain/unset.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+	"knative.dev/client-contrib/plugins/admin/pkg"
+
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
+)
+
+// updateCmd represents the update command
```
Comment (zhanggbj) [inline plugins/admin/pkg/command/domain/unset.go:29]: Could you please help to remove the commented line here? Thanks!

**Comment by zhanggbj** [comment]
@ZhuangYuZY 
The PR looks good! And I think it's good to have the following updates with this PR.  

1. The help massage and example in kn-admin [README] about unset.(https://github.com/knative/client-contrib/blob/master/plugins/admin/README.adoc)

File: plugins/admin/pkg/command/domain/unset.go (hunk 2 above)
Comment (ZhuangYuZY) [reply plugins/admin/pkg/command/domain/unset.go:29]: change is done

File: plugins/admin/pkg/command/domain/domain.go (hunk 1 above)
Comment (ZhuangYuZY) [reply plugins/admin/pkg/command/domain/domain.go:27]: change is done

**Comment by ZhuangYuZY** [comment]
@zhanggbj refining is done based on your comment. Thank you.

**Comment by zhanggbj** [comment]
@ZhuangYuZY Great and Merged 👍
